<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableToolDialog</class>
 <widget class="QDialog" name="TableToolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert Table</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="rowsLabel">
     <property name="text">
      <string>Rows:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="rowsSpinBox">
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="columnsLabel">
     <property name="text">
      <string>Columns:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="columnsSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="tableLabel">
     <property name="text">
      <string>Table:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="tableGroupBox">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="tableGridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="headersLabel">
        <property name="text">
         <string>Headers</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="alignmentLabel">
        <property name="text">
         <string>Alignment</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TableToolDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TableToolDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rowsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TableToolDialog</receiver>
   <slot>tableSizeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>columnsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TableToolDialog</receiver>
   <slot>tableSizeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tableSizeChanged()</slot>
 </slots>
</ui>
